Marlin/Marlin_main.cpp

```diff
@@ -6964,6 +7032,7 @@ void disable_all_steppers() {
 void idle() {
   manage_heater();
   manage_inactivity();
+  host_keepalive();
```

Comment:
I might just be blind and missing something obvious here, but how exactly does `idle` differ from `idle2` if it also includes `host_keepalive`?